src/main/java/org/weakref/jmx/ManagedAttribute.java

```diff
@@ -0,0 +1,56 @@
+/**
+ *  Copyright 2022 Matt Stephenson
```

Comment:
Remove the copyright line. It won't be accurate as this class evolves over time and it can always be inferred from the change history.